website/docs/r/cognito_user_pool.markdown

```diff
@@ -126,6 +126,7 @@ The following additional attributes are exported:
 
 * `id` - The id of the user pool.
 * `arn` - The ARN of the user pool.
+* `endpoint` - The endpoint name of the user pool.
```

Comment:
Since the Cognito documentation does not signify what this attribute's value might be we should include an example of what is meant here, e.g. `Example: cognito-idp.us-east-1.amazonaws.com/xxxx_yyyyy`